Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsFragment.java

```diff
@@ -89,20 +89,6 @@ public void onAttach(Context context)
     mPresenter.onAttach();
   }
 
-  /**
-   * This version of onAttach is needed for versions below Marshmallow.
```

Comment:
We still have minSdkVersion set to 21. Will this fragment continue to work on old versions of Android despite what this comment says?

Reply:
This method is now deprecated and replaced by `void onAttach(@NonNull Context context)`

Reply:
And that means that it will work on older versions?

Reply:
Yes